package.json

```diff
@@ -7,7 +7,8 @@
     "start": "node index",
     "dev": "./node_modules/.bin/nodemon -L -e js -i node_modules -i .git -V index.js",
     "lint": "./node_modules/.bin/eslint ./ --ext .js",
-    "test": "./node_modules/.bin/jest --config jest.config.json"
+    "test": "./node_modules/.bin/jest --config jest.config.json",
+    "lint:fix": "eslint . --fix"
```

Comment:
calling `eslint` requires the global dependency to be installed, so use `./node_modules/.bin/eslint` instead (see the `lint` npm script)